docs/bot/systemd.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 4
+sidebar_label: Installation with Docker and systemd
+---
+
+# Installation with Docker and systemd
+
+If docker is installed on your system, installing Draupnir is as
+simple as creating the following systemd unit file.
+
+:::warning[TODO]
+
+Questions:
+
+1. Do we need a specific user to write to the data directory? Does
+docker even work well with that?
+
+:::
+
+## Pre-requisites
+
+This guide assumes you have read over the different options available
+for starting Draupnir and understand the requirements Draupnir
+has, which you can read [here](./starting_draupnir).
+
+## Preparing your system
+
+The first thing that you will need to do is create the directory
+`/var/lib/draupnir`, which will be used to store Draupnir's
+persistent data.
+
+You will then want to copy your configuration file for Draupnir to
+`/var/lib/draupnir/config/production.yaml`.
+
+Then you will want to copy the following systemd unit file to
+`where?` and call it `draupnir.service`.
```

Comment:
Where are we copying the service file to? Well if we're storing persistent data in `/var/lib/draupnir` then we should probably just make it a system service file.